build.gradle

```diff
@@ -14,8 +15,25 @@ compileJava.options.encoding = 'UTF-8'
 group = pluginGroup
 version = pluginVersion
 
+repositories {
+    maven {
+        name = "aikar"
+        url = "http://ci.emc.gs/nexus/content/groups/aikar"
+    }
+}
+
+configurations {
+    shade
+
+    compile.extendsFrom shade
+}
+
+shadowJar.configurations = [project.configurations.shade]
+
 dependencies {
     compile 'org.spongepowered:spongeapi:5.1.0-SNAPSHOT'
+
+    shade "co.aikar:taskchain-sponge:3.4.3"
 }
 
```

Comment:
Instead of changing the build statement required, couldn't we add something like the following to the build.gradle? I am not too experienced with Gradle so I'm not sure.

```
shadowJar {
    dependencies {
        include dependency('co.aikar:taskchain-sponge')
    }
}
```